.github/workflows/main.yml

```
@@ -177,18 +175,19 @@ jobs:
         run: |
           coverage3 combine
           mv .coverage ./ci-artifact-data/fin.dat
-        if: ${{ matrix.python-version == 3.7 }}
+        if: ${{ matrix.python-version == 3.8 }}
```

```suggestion
        if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version == 3.8 }}
```
Related to the `run-tests` change